types/bun/package.json

```diff
@@ -6,7 +6,7 @@
         "https://bun.sh"
     ],
     "dependencies": {
-        "bun-types": "1.1.9"
+        "bun-types": "1.1.10"
```

Comment:
please update package version as well

Reply:
This is a patch, why do I need to bump package version?